api/konnect/v1alpha1/konnect_entity_status.go

```diff
@@ -108,16 +116,18 @@ type KonnectEntityStatusWithControlPlaneAndServiceRefs struct {
 	ServiceID string `json:"serviceID,omitempty"`
 }
 
-// SetServerURL sets the server URL of the KonnectEntityStatus struct.
+// SetControlPlaneID sets the server URL of the KonnectEntityStatus struct.
 func (in *KonnectEntityStatusWithControlPlaneAndServiceRefs) SetControlPlaneID(id string) {
```

Comment:
I was trying to find a linter recently which would find those copy paste errors but I've failed 😢 

I believe we should try to either find one (which I might have missed in my search) or write one to enforce godoc comments format.